frontend/src/metabase/ui/components/inputs/Textarea/Textarea.styled.tsx

```diff
@@ -9,16 +8,5 @@ export const getTextareaOverrides = (): MantineThemeOverride["components"] => ({
       minRows: 2,
       maxRows: 6,
     },
-    styles: (theme, _, { size = "md" }) => ({
-      wrapper: {
-        "&:not(:only-child)": {
-          marginTop: theme.spacing.xs,
-        },
-      },
-      label: {
-        color: theme.fn.themeColor("text-medium"),
-        fontSize: getSize({ size, sizes: theme.fontSizes }),
-      },
-    }),
```

Comment:
Same question here about `color` and `fontSize`

Reply:
Same answer!